packages/blockchain-wallet-v4/src/redux/wallet/sagas.ts

```diff
@@ -103,6 +104,10 @@ export default ({ api, networks }) => {
       networks.btc
     )
     yield call(api.createWallet, email, captchaToken, wrapper, forceVerifyEmail)
+    // insert api to set referral code
+    if (referral) {
+      // yield call()
+    }
```

Comment:
is this by intention? if we do not need it maybe to remove?

Reply:
oops good 👀